packages/components/CHANGELOG.md

```diff
@@ -20,14 +20,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ```diff
 - <p-fieldset-wrapper label="Some legend label">
--   <p-text-field-wrapper label="Some label">
--     <input type="text" name="some-name" />
--   </p-text-field-wrapper>
+    <p-text-field-wrapper label="Some label">
```

Comment:
Shouldn't the `-/+` part just be in the first and last line? Everything else is highly redundant. 

```diff
- <p-fieldset-wrapper label="Some legend label">
+ <p-fieldset label="Some legend label">
    <p-text-field-wrapper label="Some label">
      <input type="text" name="some-name" />
    </p-text-field-wrapper>
- </p-fieldset-wrapper>
+ </p-fieldset>
```